src/library_webgpu.js

```diff
@@ -25,19 +25,14 @@
   // Helper functions for code generation
   global.gpu = {
     makeInitManager: function(type) {
-      var mgr = 'WebGPU.mgr' + type
-      return mgr + ' = ' + mgr + ' || new Manager();';
+      var mgr = `WebGPU.mgr${type}`;
+      return `${mgr} = ${mgr} || new Manager();`;
```

Comment:
Is there some reason for the conditional initialization here?   Why not just always create a new Manager?

Reply:
I'm honestly not sure. Likely in some old version of the code, we would attempt to initialize more than once, but I don't know that we don't do that now.

FYI I might get a chance do to a major overhaul on these bindings, and at that point would like to something more generic instead of this pile of managers, so I'm not worried right now about seeing if we can clean this up.